example/index.html

```diff
@@ -30,16 +30,16 @@ <h2>Scope</h2>
     <ul>
       <li><cite id="bib-ietf-rfc-3986">IETF RFC 3986</cite>, Uniform Resource
       Identifiers (URI): Generic Syntax, T. Berners-Lee, et al., January 2005.
-      http://www.ietf.org/rfc/rfc3986.txt</li>
+      <a href="http://www.ietf.org/rfc/rfc3986.txt">http://www.ietf.org/rfc/rfc3986.txt</a></li>
```

Comment:
```suggestion
      url: <a href="http://www.ietf.org/rfc/rfc3986.txt">http://www.ietf.org/rfc/rfc3986.txt</a></li>
```

Reply:
addressed in https://github.com/SMPTE/html-pub/pull/12/commits/18c2b626d8a92c1fbfc7fd5faf76495e8cde912e, had to also add a `<br>` to force the URL to new line at end